easybuild/easyconfigs/b/bcl2fastq2/bcl2fastq2-2.20.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'bcl2fastq2'
+version = '2.20.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html'
+description = """bcl2fastq Conversion Software both demultiplexes data and converts BCL files generated by
+ Illumina sequencing systems to standard FASTQ file formats for downstream analysis."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://webdata2:webdata2@ussd-ftp.illumina.com/downloads/software/bcl2fastq/']
+sources = [{
+    'filename': '%s-v%s-tar.zip' % (name, version.replace('.', '-')),
+    'extract_cmd': 'unzip -p %s | tar -xzvf -',  # source file is a .zip that contains a .tar.gz
+}]
+
+checksums = ['8dd3044767d044aa4ce46de0de562b111c44e5b8b7348e04e665eb1b4f101fe3']
+
+start_dir = 'src'
+configopts = '--force-builddir'
```

Comment:
@SethosII Style nitpicking (which we really should automate...): please move these two lines down below `dependencies`

Reply:
@boegel I changed this.